clinica/iotools/converters/adni_to_bids/adni_utils.py

```diff
@@ -721,9 +721,8 @@ def create_adni_sessions_dict(
     # Nv/None refer to sessions whose session is undefined. "sc" is the screening session with unreliable (incomplete)
     # data.
     df_subj_session = df_subj_session[
-        (~df_subj_session.session_id.isin(["ses-Nv", "sc", None]))
+        df_subj_session.session_id.str.contains("ses-M\d*")
```

Comment:
Change to 1 or more digit:

```suggestion
        df_subj_session.session_id.str.contains("ses-M\d+")
```